### DISCUSSION TITLE
Increase cadence of version releases

### DISCUSSION THREAD

**Discussion by fbrodrigorezino** [python-poetry/poetry#8143] [Ideas]
Hi! 
   I want to suggest a faster cadence of release. Poetry has a monthly (almost) patch version, but we have several commits with very interesting and important things that keep hanging for quite some time. Can you consider changing it? 
  Maybe a regular version generation per PR merged. So we are always up-to-date with new things. 

Thank you!

**Comment by radoering** [comment]
I don't think this will happen. Each release costs some effort. It's not just preparing the release but also increased support effort after the release. There is just not enough maintainer time to handle this in a faster cadence.

If you want to be always up to date with new things, you can install poetry from its master branch. You can also use the hash from a specific commit if you want to avoid unexpected changes that may break your CI. See the [docs](https://python-poetry.org/docs/#installing-with-pipx) for more information on how to install poetry from git.

**Reply by fbrodrigorezino** [reply]
> If you want to be always up to date with new things, you can install poetry from its master branch. You can also use the hash from a specific commit if you want to avoid unexpected changes that may break your CI. See the [docs](https://python-poetry.org/docs/#installing-with-pipx) for more information on how to install poetry from git.


It can work well for particular projects, but working around it is not welcome in enterprise environments. 

I understand it might be complex and maybe not possible, which can become a point against adopting poetry. So customers need to choose, but it's the game, and I totally understand it. 
For instance, I'm having problems with Poetry 1.5.1 and private repositories, so we are not using it and waiting to test the next release. But our last Patch release was two months ago, on 29 April. This caused a lot of problems internally already, mainly because the private repositories configuration has breaking changes from 1.3, 1.4, 1.5. So we are in this void of problems, asking developers not to update and keep in a version we know things are working, changing CI/CD scripts everywhere, summarizing, and many costs. When a new version is released, we will need to revert everything. 

But, ok, I got it, I will take this perception internally, and we will see what to do given this view. 

Thanks for the feedback.
